src/xdocs/config_coding.xml

```diff
@@ -4953,6 +4953,73 @@ if (&quot;something&quot;.equals(x))
       </subsection>
     </section>
 
+    <section name="UnnecessarySemicolonAfterTypeMemberDeclaration">
+      <p>Since Checkstyle 8.24</p>
+      <subsection name="Description"
+                  id="UnnecessarySemicolonAfterTypeMemberDeclaration_Description">
+        <p>
+          Checks if unnecessary semicolon is used after code type member declaration.
+          The check will flag the following with warnings:
+        </p>
+        <source>
+class A {
```

Comment:
These seem better suited in the `Examples` section then here. Please move them there and add the words violation.
Example: `violation - standalone semicolon`.
Example: https://checkstyle.sourceforge.io/config_naming.html#AbbreviationAsWordInName_Examples

Reply:
done